homeassistant/components/hvv_departures/binary_sensor.py

```diff
@@ -0,0 +1,195 @@
+"""Binary sensor platform for hvv_departures."""
+from datetime import timedelta
+import logging
+
+from aiohttp import ClientConnectorError
+import async_timeout
+from pygti.exceptions import InvalidAuth
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import ATTRIBUTION, CONF_STATION, DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the binary_sensor platform."""
+    hub = hass.data[DOMAIN][entry.entry_id]
+    station_name = entry.data[CONF_STATION]["name"]
+    station = entry.data[CONF_STATION]
+
+    def get_elevator_entities_from_station_information(
+        station_name, station_information
+    ):
+        """Convert station information into a list of elevators."""
+        elevators = {}
+
+        if station_information is None:
+            return {}
+
+        for partial_station in station_information.get("partialStations", []):
+            for elevator in partial_station.get("elevators", []):
+
+                state = elevator.get("state") != "READY"
+                available = elevator.get("state") != "UNKNOWN"
+                label = elevator.get("label")
+                description = elevator.get("description")
+
+                if label is not None:
+                    name = f"Elevator {label} at {station_name}"
+                else:
+                    name = f"Unknown elevator at {station_name}"
+
+                if description is not None:
+                    name += f" ({description})"
+
+                lines = elevator.get("lines")
+
+                idx = f"{station_name}-{label}-{lines}"
```

Comment:
Does it make a difference if the id is "STATION_NAME-LABEL"  instead of "STATION_NAME-LABEL-**None**"?
If the lines key exist it differenciates the elevators, and if it is not available the label key should be unique (which is what I observed from using the api, again both keys are marked as optional in the docs).

Reply:
Ok.